src/develop/imageop.c

```diff
@@ -3045,6 +3050,50 @@ void dt_iop_cancel_history_update(dt_iop_module_t *module)
   }
 }
 
+char *dt_iop_set_description(dt_iop_module_t *module, const char *main_text, const char *purpose, const char *input, const char *process,
+                             const char *output)
+{
+  char *str_purpose = g_strdup(_("purpose"));
```

Comment:
all those strings can just be:

`const char *str_purpose = _("purpose");`

Reply:
done